circle.yml

```diff
@@ -7,14 +7,15 @@ machine:
   environment:
     _JAVA_OPTIONS: "-Xmx512m"
     GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx512m -XX:+HeapDumpOnOutOfMemoryError"'
+    TERM: dumb
   services:
     - docker
 
 dependencies:
   pre:
     - sudo pip install docker-compose
   override:
-    - export TERM="dumb"; ./gradlew compileTestJava
+    - ./gradlew compileTestJava --console=plain --stacktrace
```

Comment:
Nit: might be worth extracting this into an environment variable (`GRADLE_ARGS`?)